end-to-end-tests/imageCompare.html

```diff
@@ -1,8 +1,8 @@
 <html>
     <head>
         <script src="https://code.jquery.com/jquery-latest.min.js"></script>
-        <script src="image-compare/errors.js"></script>
-        <script src="image-compare/js/main.js"></script>
+        <script src="https://2600-66571349-gh.circle-artifacts.com/0/image-compare/errors.js"></script>
```

Comment:
should this point to relative `errors.js`?

Reply:
nice catch